src/modules/fancyzones/FancyZonesLib/MonitorUtils.cpp

```diff
@@ -369,10 +355,20 @@ namespace MonitorUtils
 
         auto displays = Display::GetDisplays();
         auto monitors = WMI::GetHardwareMonitorIds();
+
+        // retry 
+        int retryCounter = 0;
+        while (!displays.has_value() && retryCounter < 100)
```

Comment:
I wonder if 100 is a bit too much? With 100 milliseconds sleep this can take 10 seconds right? This is to much :) Do you have info after how many retries usually it returns correct result?

Reply:
I thought about 10 at first, 20-30 at max, but I decided to play it safe and set 100 just to check how many would appear. But then I saw 48 retries in the report. A half! So maybe it's not THAT much :)

Reply:
in that case, 100 is a reasonable number, but together with 100ms sleep, it can take up to 10 seconds in the worst case scenario and that is too much imho. Maybe at least to lower the sleep time. wdyt?

Reply:
I agree, reducing overall waiting time makes sense. Do you think that 10ms would be ok?